pages/common/repren.md

```diff
@@ -0,0 +1,23 @@
+# repren
+
+> Multi-pattern string replacement and file renaming tool.
+
+- Do a dry-run renaming a folder of pngs with a literal string replacement:
+
+`repren --dry-run --literal --from '{{find_string}}' --to '{{replacement_string}}' *.png`
+
+- Do a dry-run renaming a folder of jpegs with a regular expression:
+
+`repren --rename --dry-run --from '{{regular_expression}}' --to '{{replacement_string}}' *.jpg *.jpeg`
+
+- Do a find-and-replace on the contents of a folder of csv files:
+
+`repren --from '{{([0-9]+) example_string}}' --to '{{replacement_string \1}}'`
```

Comment:
Shouldn't there be a ".csv" extension at the end ?

Reply:
Ah, yeah! I must have forgotten :P